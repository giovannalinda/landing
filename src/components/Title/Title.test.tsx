import { render, screen } from '~/__helpers__/app-tests'
import { Title } from '.'

describe('Title', () => {
  it('should render correctly', async () => {
    render(<Title>Example</Title>)

    expect(await screen.findByText(/Example/i)).toBeInTheDocument()
  })

  it('should match snapshot', async () => {
    const { container } = render(<Title>Example</Title>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
