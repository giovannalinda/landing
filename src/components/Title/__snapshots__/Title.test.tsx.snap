// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match snapshot 1`] = `
<h1
  class="sc-bczRLJ jSHJYa"
>
  Example
</h1>
`;
